<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QWidget" name="widget_13" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QWidget" name="widget_8" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>接收区：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="re_text"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_12" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="clear_re_but">
               <property name="text">
                <string>清空接收区</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>串口选择：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="mycomboBox" name="com_combox"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>波特率：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>停止位：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>数据位：</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>校验位：</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QComboBox" name="btl_combox">
                  <item>
                   <property name="text">
                    <string>9600</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14400</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19200</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="stop_combox">
                  <item>
                   <property name="text">
                    <string>1</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>1.5</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>2</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="data_combox">
                  <item>
                   <property name="text">
                    <string>8</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>7</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>6</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>5</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="jiaoyan_combox">
                  <item>
                   <property name="text">
                    <string>None</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Odd</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Even</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="hex_send_cheak">
               <property name="text">
                <string>hex发送</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="hex_re_cheak">
               <property name="text">
                <string>hex接收</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_15" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QCheckBox" name="gbk_re">
               <property name="text">
                <string>GBK接收</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="gbk_send">
               <property name="text">
                <string>GBK发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>串口操作：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="open_radiobut">
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>   发送区：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_14" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTextEdit" name="send_text"/>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="send_but">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clear_send_but">
               <property name="text">
                <string>清除发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mycomboBox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
